python/src/spark_rapids_ml/tree.py

```diff
@@ -148,8 +154,57 @@ class _RandomForestCumlParams(
     HasFeaturesCols,
     HasLabelCol,
 ):
+
+    n_streams = Param(
```

Comment:
Are these converted to Params to enable modification on copy?   For spark.mllib estimators I think ok to do that only for mapped params.   Or at least is a separate topic.

Reply:
Yeah, this is to avoid changing function signature: def copy(self: P, extra: Optional["ParamMap"] = None) -> P, and ParamMap = Dict[pyspark.ml.param.Param, Any]. 

copy() only accepts Param type.

Reply:
Sorry if I'm missing something -  how come we need to declare RF cuml params as Params here and not in other algos?

Reply:
Right. We will need to add the cuml-only params (e.g. `whiten` and `svd_solver`) one by one as Param, in order to get them working properly.